cogs/study.py

```diff
@@ -52,6 +53,26 @@ async def ask_question(self, ctx, conversation_mode="true", *, question: str):
 
         await ctx.send(embed=embed)
 
+    @commands.command(name='urban')
+    async def _urban(self, ctx, *, word):
```

Comment:
I wonder, is `urban` reserved name or why did you use `_urban` there instead of just going with `urban`

Reply:
solved that, hence it's done and removed paginator too